dashboard/src/components/Table/BaseTable.tsx

```diff
@@ -28,7 +34,7 @@ const BaseTable = ({ headers, body, className }: IBaseTable): JSX.Element => {
             ))}
           </TableRow>
         </TableHeader>
-        {body}
+        {body || children}
```

Comment:
this should reflect on the type
you can use `never` for this case

I'm pretty sure is something like 

`& { body: React.ReactNode, children: never } | { children: React.ReactNode, body: never }